Vagrantfile

```diff
@@ -85,6 +85,9 @@ Vagrant.configure("2") do |config|
     destination: ".ssh/#{devconfig['ssh_key']}.pub"
 
   config.vm.provision "shell", path: "config/provision_vagrant.sh"
+  config.vm.provision "docker" do |d|
+    d.run "redis", image: "redis:latest", args: "-p 6379:6379", daemonize: true
```

Comment:
We don't need Docker to provision Redis. I'd much rather just install the `redis-server` package.